.github/workflows/integration.yml

```diff
@@ -0,0 +1,23 @@
+
+on: push
```

Comment:
Shouldn't this be part other docker workflow?  I think that workflow should first run these steps to build and test the container (on each commit) and then push the tested container to the repo (on release commits only).